app/templates/components/wds-global-navigation/search.hbs

```diff
@@ -1,33 +1,49 @@
-{{#wds-dropdown class=(concat 'wds-global-navigation__search-input-wrapper' (if hasSuggestions ' wds-is-active')) as |dropdown|}}
-	{{#dropdown.toggle tagName='label' class='wds-global-navigation__search-label'}}
-		{{wds-icon name='magnifying-glass' size='small' class='wds-global-navigation__search-label-icon'}}
+{{#wds-dropdown dropdownExpanded=hasSuggestions class='wds-global-navigation__search' as |dropdown|}}
+	<div class="wds-global-navigation__search-toggle" {{action 'openSearch'}}>
```

Comment:
This element has no role making it harder/impossible to use with assistive technology: https://github.com/evcohen/eslint-plugin-jsx-a11y/blob/master/docs/rules/no-static-element-interactions.md